### DISCUSSION TITLE
How to delete points in an Image?

### DISCUSSION THREAD

**Discussion by xxmd1132** [wkentaro/labelme#1131] [Ideas / Feature requests]
Hi,when use labelme,I find a an unfriendly design.I can not delete point  after create a new point.And I can know  straightly which point is selected in an image.I think labelme can modify according to method that labelimg delete a rect(select rect and Press DEL from keyboard ).

**Comment by wkentaro** [comment]
I'm not sure what do you mean, but maybe you want to delete point-annotation instead of points from polygon-annotation?

**Comment by yuhorun** [comment]
> I'm not sure what do you mean, but maybe you want to delete point-annotation instead of points from polygon-annotation?

她的意思是  在标注point的状态下 不能直接删除已标注point   要先到多边形标注状态下  再选择编辑多边形  才能去删除point

**Comment by valleu326** [comment]
as #399，we can delete shapes like polygon, circle, rectangle, but can not delete point or line, it is unreasonable if we can only create point/line but can not delete them.

**Comment by ChangXuSunny** [comment]
I encounter same problem. I draw a line. But when I want to delete, the function delete polygon doesn't work for line. Help needed! Thanks

**Comment by xxmd1132** [comment]
You can update labelme to version 4.2.9.the author has added the function which is required.thks author again～～



---Original---

**Comment by idkondor** [comment]
Hello everybody. Is there in version 4.2.9 of Labelme, possibility  to delete several points and not just only one?

**Comment by zherunli** [comment]
Hey guy. It seems like we cannot directly delete the points that are created in the UI. But it is not impossible to delete the points. We can edit the corresponding json file and, simply detele the part that defines the very point.

Hope it helpes.
